models/articles.models.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.selectAllArticles = async (article_id) => {
```

Comment:
This function should now no longer be called `selectAllArticles` - what should it be?